### DISCUSSION TITLE
Change background of window

### DISCUSSION THREAD

**Discussion by ghost** [joeattardi/picmo#251] [Q&A]
Its possible customize the window/frame background using css.

Thanks

**Comment by joeattardi** [comment]
Do you mean the picker background color?

This can be done with theming. See [here](https://picmojs.com/docs/usage/themes) for details.

**Reply by ghost** [reply]
change from white to site background rgba pattern

**Reply by joeattardi** [reply]
I don't understand what you're asking. What white background color are you trying to change? Screenshots might help.
